app.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const app = express();
+const { handle404 } = require('./errorHandle');
+const { getTopics } = require('./controller/topicController');
+
+app.use(express.json());
+
+app.get('/api/topics', getTopics);
+
+app.use(handle404);
```

Comment:
Its great to see a 404 error handler here, but any code you add should also be tested for and you dont currently have any tests to ensure this works as we might expect it to.

How would you test to ensure this works as we expect it to?